packages/forms-web-app/src/pages/projects/register/index/_utils/is-registration-open.js

```diff
@@ -0,0 +1,17 @@
+const dayjs = require('dayjs');
+const isSameOrAfter = require('dayjs/plugin/isSameOrAfter');
+
+dayjs.extend(isSameOrAfter);
+
+const isRegistrationOpen = (openDate, closeDate) => {
+	const dayToday = dayjs();
+	const registrationOpenDate = dayjs(openDate);
+	const registrationCloseDate = dayjs(closeDate);
```

Comment:
shouldn't registration stay open until the close date 23:59:59?
eg: `const registrationCloseDate = dayjs(closeDate).endOf('day');`

I might be wrong, not sure how the registration should work, ignore if it's meant to close on the day